src/main/scala/gitbucket/core/util/Implicits.scala

```diff
@@ -73,7 +73,7 @@ object Implicits {
 
     def hasAttribute(name: String): Boolean = request.getAttribute(name) != null
 
-    def gitRepositoryPath: String = request.getRequestURI.replaceFirst("^/git/", "/")
+    def gitRepositoryPath: String = request.getRequestURI.replaceFirst("^" + request.getContextPath + "/git/", "/")
```

Comment:
contextPath can contain some special chars ( for example, dot). you should escape contextPath. maybe you can use `java.util.regex.Pattern.quote`.